src/main/java/org/roy/loadx/transaction/TransactionPrinter.java

```diff
@@ -81,6 +81,16 @@ private String getTransactionDisplayName(String transactionName) {
 		}
 	}
 	
+	static private void println() {
+		System.out.println();
+		System.out.flush();
+	}
+	
+	static private void println(Object obj) {
+		System.out.println(obj);
+		println();
```

Comment:
Adding empty line is a logic of the client does not belong here. If it did, method printlnWithLineSeparator for example which would be confusing here.